client/src/components/AddClimbedBoulderForm.jsx

```diff
@@ -65,7 +65,9 @@ const AddClimbedBoulderForm = ({
           <ButtonWrapperLeft>
             <CounterButton
               onClick={() => {
-                setNewAttempts(newAttempts - 1);
+                setNewAttempts(
+                  newAttempts === 0 ? newAttempts : newAttempts - 1
+                );
```

Comment:
Maybe it could be more safe if you ask like this:

`newAttempts > 0 ? newAttempts -1 : newAttempts`

This would also include negative cases, even if they are unlikely :P

Reply:
Cool. I have changed this accordingly.